commander/src/commands/account/show.ts

```diff
@@ -22,14 +26,24 @@ import { getPassphraseFromPrompt } from '../../utils/reader';
 
 const processInput = (
 	passphrase: string,
-): { privateKey: string; publicKey: string; address: string } => {
+): {
+	privateKey: string;
+	publicKey: string;
+	address: string;
+	binaryAddress: string;
+} => {
 	const { privateKey, publicKey } = getKeys(passphrase);
-	const address = getAddressFromPublicKey(publicKey);
+	const binaryAddress = getAddressFromPublicKey(publicKey);
+	const address = getBase32AddressFromPublicKey(
+		publicKey.toString('hex'),
+		'lsk',
+	);
 
 	return {
-		privateKey,
-		publicKey,
+		privateKey: privateKey.toString('base64'),
+		publicKey: publicKey.toString('base64'),
 		address,
+		binaryAddress: binaryAddress.toString('base64'),
```

Comment:
Why do we need binary address in the output. And binary address in `base64` encoding makes it confusing. 

That' the output of the `account:create` command. I suggest only `address` is good enough to show there. 

```
[{
"passphrase":"because hunt remember blue ship close frog reveal also jaguar fiscal kidney",
"privateKey":"03SbeIOUEYOqPlJyNuMoHi4mXPrI3vYmm5xXUKuGrwlWHKCElmGwM+4syf+OaUgS5CgyswOzUhCpBnGpRuRpXA==",
"publicKey":"VhyghJZhsDPuLMn/jmlIEuQoMrMDs1IQqQZxqUbkaVw=",
"binaryAddress":"ScjH0f/rz4MOAB16P7CvYyvAB7Y=",
"address":"lsk4npo8k8gh8n2bcuzcqhcgom8ooqjzxhe3m9mj3"}]
```

Reply:
If all the endpoint from the framework uses base64, wouldn't it be useful to know the binary address in base64 format?
At least while fixing the test, it was useful

Reply:
I would suggest, outside from framework/application using any interface http/api we use only base64 address not base64 encoding of binary address. Binary address should stay under layers of libraries and framework.